common-custom-user-data-maven-extension/src/main/java/com/gradle/ApiAccessor.java

```diff
@@ -10,31 +9,28 @@
 
 import static java.util.Comparator.comparing;
 
-final class BuildScanApiAccessor {
+final class ApiAccessor {
```

Comment:
This class is referenced in the Maven extension documentation in the programmatic configuration section. We need to adjust that section.

Reply:
Yep nice catch. Marc already mentioned this on the other pull request and there is an after release step activity to update it.